docs/remote/containers-advanced.md

```diff
@@ -57,7 +57,7 @@ Next, depending on what you reference in `devcontainer.json`:
 * **Dockerfile or image**: Edit `devcontainer.json` and add a path to the `.env` file relative to the location of `devcontainer.json`:
 
     ```json
-    "runArgs": ["--env-file","devcontainer.env"]
+    "runArgs": ["--env_file","devcontainer.env"]
```

Comment:
The docker run switch is actually --env-file. It is only dockercompose which is env_file.
I'll revert this change.